### ISSUE TITLE
error in the path not found

### ISSUE THREAD

**Issue by Daniel7s20** [Falcosc/enable-loudness-equalisation#8]
I get an error after putting the name of the speaker I get path not found, any solution?
![Annotation 2023-05-22 150211](https://github.com/Falcosc/enable-loudness-equalisation/assets/134330577/f4bfcb4a-e6af-4b06-89b9-2079f186a94e)

**Comment by Falcosc** [comment]
If there is 0 configuration there, it will not work.

You could export the whole Renderer Directory to give it a 2nd chance. Maybe you have multiple devices with this name. But if it already found the correct device, then it will not work.

**Comment by xShooerman** [comment]
> 

hey do you think theres a way with voice meeter that you know of and force it or soemthing? casue i use voice meeter and plus i use corsair HS70 wireless and i have icue and that removes it but i use everything on voice meeter for my mixer stuff. and if not do you know or have anyone know a setting with voice meeter to get it? casue they have a EQ to mess with but idk what to do.

**Comment by Falcosc** [comment]
No, you need third party apps if it isn't hidden in your audio driver.

**Comment by xShooerman** [comment]
I think I found a way for it to work. It’s working but tested on music and stuff but not fully sure yet with the loudness EQ gonna year in on MW2 soon. And with the 3rd party Ik Icue removed the tab cause of Icue is installed. But what I did is when trying to enable the eq and gave the error I followed the reg path with reg edit and made sure I had perms to overwrite it and then exported the file that was my voice meeter and used notpad++ removed everything and only had were it adds the FxProp folder and once I’ve done that and re ran the cmd to get the EQ it worked and was installed. So it’s working just not 100% on the other parts. I just know I had to add the folder to the vb cable as well and made sure it was set up right in my voice meeter and make sure my games/apps I want to use vb cable as for my listening and it worked. So idk anything else. Lmk what you think cause again I’ve got it to work
